buildifier/BUILD

```diff
@@ -0,0 +1,27 @@
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load(
+    "@buildifier_prebuilt//:rules.bzl",
+    "buildifier_binary",
+)
+
+bzl_library(
+    name = "buildifier_binary",
+    srcs = ["buildifier_binary.bzl"],
+)
+
+# Buildifier toolchain type
+toolchain_type(
+    name = "toolchain",
+    visibility = ["//visibility:public"],
+)
+
+buildifier_binary(
+    name = "buildifier",
```

Comment:
Moving this probably breaks the current usage with `//:buildifier`? I guess we should try to keep that if possible, either by keeping this in the top level or using an alias

Reply:
Doh. Sorry about that. I did not have a test for that test case. Do you have an example command line that you use with the current version of the buildifier-prebuilt? I will add tests for those and then add aliases.

Reply:
just the ones in the readme are probably fine

Reply:
I was able to implement a test for `buildifier` using the simple example workspace. However, I believe that `buildozer` will not work as described in the `README.md`. I have documented the issue in #9. Let me know what you think. I did  go ahead and add the aliases for both `buildifier` and `buildozer`.